### PR TITLE
fix: quote FILES when running shellcheck

### PR DESCRIPTION
**Pull request by folderol4**

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -55,7 +55,7 @@ echo '::group:: Running shellcheck ...'
 if [ "${INPUT_REPORTER}" = 'github-pr-review' ]; then
   # erroformat: https://git.io/JeGMU
   # shellcheck disable=SC2086
-  shellcheck -f json  ${INPUT_SHELLCHECK_FLAGS:-'--external-sources'} ${FILES} \
+  shellcheck -f json  ${INPUT_SHELLCHECK_FLAGS:-'--external-sources'} "${FILES}" \
```
Comment (folderol4) [inline script.sh]: This does better in that it seems to list out the sh scripts now with spaces, but still is failing:
image

Comment (folderol4) [reply script.sh]: https://github.com/folderol4/shellcheck-test/actions/runs/3238736808/jobs/5307275250

**Comment by grische** [comment]
@folderol4 can this PR be closed?

**Comment by javierjulio** [comment]
There's been no update in a long time and has merge conflicts now so closing out.

**Comment by folderol4** [comment]
Sounds good. I walked away from using this action because I could never get it to work correctly with multiple paths that had spaces in them.

**Pull request closed without merging**